plugins/inputs/filestat/filestat_test.go

```diff
@@ -18,7 +23,7 @@ func TestGatherNoMd5(t *testing.T) {
 	fs.Files = []string{
 		dir + "log1.log",
 		dir + "log2.log",
-		"/non/existant/file",
+		dir + "non_existent_file",
```

Comment:
You are using `dir + "non_existent_file"` multiple times in this file, maybe it's worth a variable?

Reply:
`testdataDir` is now a variable - I think it will do :)